src/indicators/ipc_food_insecurity/utils/plot_food_insecurity.R

```diff
@@ -177,7 +179,10 @@ food_insecurity_ts <- function(df_wrangled, df_raw, title, date) {
     gg$expand_limits(
       y = 0
     ) +
-    theme_signals$theme_signals("subtitle") +
+    theme_signals$theme_signals(
+      margin_location = "subtitle",
+      x_axis_ticks = TRUE
```

Comment:
Wouldn't this arg need to be added to IDMC, ADLED, and WHO plotting functions as well?

Reply:
No, because within those functions, they are calling `plot_ts()` to create the generic line plots, so we just have to adjust it in `plot_ts()`. See the `plot_displacement.R` script for example: 

https://github.com/OCHA-DAP/hdx-signals/blob/9e2a6d8e5d2928bfd6c8325fdd73ca28cdf61cf0/src/indicators/idmc_displacement/utils/plot_displacement.R#L63